source/reference/failure-recovery.txt

```diff
@@ -0,0 +1,97 @@
+.. _c2c-failure-recovery:
+
+==========================
+Recovery From Sync Failure
+==========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+:program:`mongosync` temporarily alters some collection characteristics
+during synchronization. The original values are restored during the
+:ref:`commit process <c2c-api-commit>`. However, a network failure or
+some other problem could disrupt synchronization before the sync is
+committed. If the sync ends prematurely, you must manually update the
+temporary changes on the destination cluster.
```

Comment:
"temporary changes" sounds a bit weird but I don't know what's the better wording here. "intermediate states"?